source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+---
+title: "Core Concepts"
+description: "How Home Assistant works."
+---
+
+Home Assistant, like all home automation software, allows you to control smart devices in your home. This page will explain the parts of Home Assistant and how they relate.
+
+## Devices
+
+In Home Assistant, a **device** is a physical object in your connected home. An example would be a lightbulb, a Chromecast, or a smart hub.
```

Comment:
A "smart hub" might be a bad example, as they don't have to be a device themself, however, can provide additional other devices.